app/src/test/java/nl/tudelft/b_b_w/controller/BlockControllerUnitTest.java

```diff
@@ -402,6 +406,30 @@ public void testBacktrack() {
     }
 
     /**
+     * verifyTrustworthiness test
+     * Tests whether a block is trustworthy
+     */
+    @Test
+    public void testVerifyTrustworthiness() {
+        final Block newBlock = BlockFactory.getBlock(TYPE_BLOCK, owner, bc.getLatestSeqNumber(owner)+1,
+                ownHash, previousHashChain, NA, publicKey, iban, trustValue);
+        bc.addBlock(newBlock);
+        assertTrue(bc.verifyTrustworthiness(newBlock));
+    }
+
+    /**
+     * verifyTrustworthiness test
+     * Tests whether a block is trustworthy
+     * Forces a false
+     */
+    @Test
+    public void testVerifyTrustworthinessFalse() {
+        final Block newBlock = BlockFactory.getBlock(TYPE_BLOCK, owner, bc.getLatestSeqNumber(owner)+1,
```

Comment:
The function in the BlockController is not correct, so until it has been refactored, it is discouraged to use it.